### ISSUE TITLE
Fix CardView visual issues

### ISSUE THREAD

**Issue by tasomaniac** [gdg-x/frisbee#237]
Go through all of the CardView usages and see how they look like both in portrait, landscape, phones and tablets. 
And fix visual issues like paddings, contentpaddings

**Comment by friedger** [comment]
Is a super issue containing #270 and similar

**Comment by tasomaniac** [comment]
It is not just #270 which is the EventDetail page. 
Cards are ugly in InfoFragment, EventDetail, About, Contributon, Pulse

**Comment by friedger** [comment]
What needs to be done here?

**Comment by friedger** [comment]
This can't be fixed by Release 2.2 because #270 is planned for Release 2.3. We should kill this and have more specific issues.

**Comment by tasomaniac** [comment]
I don't think it depends on #270. This issue is only about cards. 
When we started using Cards, we didn't do all of them the same. In the app currently, all the cards look different. The padding, elevation etc are all different. 
Someone should go over all the cards and ensures a consistency. I think we should reopen it.

**Comment by tasomaniac** [comment]
Lets close this becuase the app looks not bad. If we see a problem with somewhere in the app about cards, lets open another issue.
